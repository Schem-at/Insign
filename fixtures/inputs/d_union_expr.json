[
  { "pos": [0, 64, 0], "text": "@a=rc([0,0,0],[1,1,1])" },
  { "pos": [0, 64, 0], "text": "@b=rc([2,0,0],[3,1,1])" },
  { "pos": [0, 64, 0], "text": "@core=a+b\n#core:desc=\"Combined regions\"" }
]
