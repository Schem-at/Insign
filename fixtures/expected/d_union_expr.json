{
  "a": {
    "bounding_boxes": [
      [
        [
          0,
          64,
          0
        ],
        [
          1,
          65,
          1
        ]
      ]
    ],
    "metadata": {}
  },
  "b": {
    "bounding_boxes": [
      [
        [
          2,
          64,
          0
        ],
        [
          3,
          65,
          1
        ]
      ]
    ],
    "metadata": {}
  },
  "core": {
    "bounding_boxes": [
      [
        [
          0,
          64,
          0
        ],
        [
          1,
          65,
          1
        ]
      ],
      [
        [
          2,
          64,
          0
        ],
        [
          3,
          65,
          1
        ]
      ]
    ],
    "metadata": {
      "desc": "Combined regions"
    }
  }
}
